src/components/capture-area/CaptureAreaSearchValue/CaptureAreaSearchValue.vue

```diff
@@ -56,59 +98,165 @@ import {
     defaultBeforeConstraintsSettings,
 } from './ResetSettings';
 import useNotificationSystem from '@/composables/useNotificationSystem/useNotificationSystem';
+import useClipboard from '@/composables/useClipboard/useClipboard';
 
 /**
  * Definied Props
  */
-const props = defineProps<{
-    captureAreaId: number;
+const defProps = defineProps<{
+    captureAreaId: string;
 }>();
 
+/**
+ * Composables
+ */
+const { writeClipboardText } = useClipboard();
+
 /**
  * Get singelton instance reference to vigad
  */
 const vigad = ref(Vigad.getInstance());
 
+/**
+ * Rules for the capture area id input
+ */
+const captureAreaIdRules = {
+    required: (value: string) =>
+        !!value || 'An access token is required to start a session',
+    uppercase: (v: string) =>
+        /[A-Z]/.test(v) || 'Must include at least one uppercase letter',
+    lowercase: (v: string) =>
+        /[a-z]/.test(v) || 'Must include at least one lowercase letter',
+    number: (v: string) =>
+        /[0-9]+/.test(v) || 'Must include at least one number',
+    max: (v: string) =>
```

Comment:
![image](https://github.com/VisualGameData/VIGAD/assets/58397976/25cf93b6-9c99-435e-b790-96231049848e)
does not seem to apply

Reply:
What do you mean, what does not apply ?

Reply:
max length

Reply:
Yeah max lenght of 6 chars you only have 3 there